synapseAnnotations/data/common/minimal_Sage_standard.json

```diff
@@ -982,222 +982,222 @@
+        "source": "https://en.wikipedia.org/wiki/NeuN"
       },
       {
         "value": "epithelial",
-        "description": "",
-        "source": ""
+        "description": "Somatic cells that cover the surface of the body and line its cavities.",
+        "source": "https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCL_0000066"
       },
       {
         "value": "epithelial-like",
-        "description": "",
-        "source": ""
+        "description": "In cell morphology, epithelial-like cells are polygonal in shape with more regular dimensions, and grow attached to a substrate in discrete patches.",
+        "source": "https://www.thermofisher.com/us/en/home/references/gibco-cell-culture-basics/cell-morphology.html"
       },
       {
         "value": "fibroblast",
-        "description": "",
-        "source": ""
+        "description": "A connective tissue cell which secretes an extracellular matrix rich in collagen and other macromolecules. Flattened and irregular in outline with branching processes; appear fusiform or spindle-shaped.",
+        "source": "https://www.ebi.ac.uk/ols/ontologies/cl/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCL_0000057"
       },
       {
         "value": "round",
-        "description": "",
-        "source": ""
+        "description": "A phenotype observation at the level of the cell shape where the cell is round",
+        "source": "https://www.ebi.ac.uk/ols/ontologies/cmpo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fcmpo%2FCMPO_0000118"
       },
       {
         "value": "lymphoblast",
-        "description": "",
-        "source": ""
+        "description": "Often referred to as a blast cell. Unlike other usages of the suffix -blast a lymphoblast is a further differentiation of a lymphocyte, T- or B-, occasioned by an antigenic stimulus. The lymphoblast usually develops by enlargement of a lymphocyte, active re-entry to the S phase of the cell cycle, mitogenesis and production of much m-RNA and ribosomes.",
+        "source": "https://www.ebi.ac.uk/ols/ontologies/bto/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FBTO_0000772"
       },
       {
         "value": "polygonal",
-        "description": "",
-        "source": ""
+        "description": "A polygonal face is a polygon bounded by a circuit of polygon edges, and includes the flat (plane) region inside the boundary.",
+        "source": "https://www.ebi.ac.uk/ols/ontologies/sio/terms?iri=http%3A%2F%2Fsemanticscience.org%2Fresource%2FSIO_000503"
       },
       {
         "value": "CD8+ T-Cells",
-        "description": "",
-        "source": ""
+        "description": " Is a T lymphocyte (a type of white blood cell) that kills cancer cells, cells that are infected (particularly with viruses), or cells that are damaged in other ways.",
+        "source": "https://en.wikipedia.org/wiki/Cytotoxic_T_cell"
       },
       {
         "value": "primary arachnoid",
```

Comment:
could you remove the whole primary arachnoid value? I am making a pull request for an isPrimary boolean which will make this redundant (I should have done that in the first place instead of adding this cellType, sorry)